TOC.md

```diff
@@ -538,6 +538,8 @@
 + 版本发布历史
   + [发布版本汇总](/releases/release-notes.md)
   + [产品路线图](/roadmap.md)
+  + v5.1
+    - [5.1](/releases/release-5.1.md)
```

Comment:
Would it be better to change the file name to `release-5.1.0.md`? I guess v5.1 might also have patch releases...